packages/devextreme/js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -411,6 +411,7 @@ const DropDownEditor = TextBox.inherit({
         this._refreshValueChangeEvent();
         this._refreshEvents();
         this._refreshEmptinessEvent();
+        this._setDefaultAria();
```

Comment:
We implemented the code for dropDownEditor, but the current test is designed for selectBox. While we can use the selectBox test to validate the customer's scenario, we should also create a test specifically for dropDownEditor